discovery-infra/tests/base_test.py

```diff
@@ -42,13 +42,15 @@ def cluster(self, api_client, request):
         clusters = []
 
         def get_cluster_func(cluster_name: Optional[str] = None,
-                             additional_ntp_source: Optional[str] = consts.DEFAULT_ADDITIONAL_NTP_SOURCE):
+                             additional_ntp_source: Optional[str] = consts.DEFAULT_ADDITIONAL_NTP_SOURCE,
+                             openshift_version: Optional[str] = env_variables['openshift_version']):
```

Comment:
This will break all the other tests cause the default of env_variables['openshift_version'] is
 "openshift_version": utils.get_openshift_version()
conftest need to have default:
utils.get_env('OPENSHIFT_VERSION', utils.get_openshift_version()),

Reply:
`get_openshift_version` function have `OPENSHIFT_VERSION` reference.

Reply:
ho right, missed that